pkg/network/node/node.go

```diff
@@ -450,6 +450,18 @@ func (node *OsdnNode) Start() error {
 // attached to the OVS bridge before restart, and either reattaches or kills each of the
 // corresponding pods.
 func (node *OsdnNode) reattachPods(existingPodSandboxes map[string]*kruntimeapi.PodSandbox, existingOFPodNetworks map[string]podNetworkInfo) error {
+	fieldSelector := fields.Set{"spec.nodeName": node.hostName}.AsSelector()
+	opts := metav1.ListOptions{
+		LabelSelector: labels.Everything().String(),
+		FieldSelector: fieldSelector.String(),
+	}
+	pods, err := node.kClient.CoreV1().Pods(metav1.NamespaceAll).List(context.TODO(), opts)
```

Comment:
hm... yeah, I know I said "just call that", but I forgot you needed to set up the fieldSelector too...

I think you could just call GetRunningPods(). It does a little bit more filtering than this, but filtering the list down to only pods that are still running is probably a good thing, since pods that aren't `Running` aren't going to have sandboxes to reattach...